chain/chain/src/chain.rs

```diff
@@ -2357,6 +2357,30 @@ impl Chain {
     pub fn is_chunk_orphan(&self, hash: &CryptoHash) -> bool {
         self.blocks_with_missing_chunks.contains(hash)
     }
+
+    /// Check if can sync with sync_hash
+    pub fn check_sync_hash_validity(&mut self, sync_hash: &CryptoHash) -> Result<bool, Error> {
+        let head = self.head()?;
+        match self.get_block_header(sync_hash) {
```

Comment:
Can just replace this match with `let sync_header = self.get_block_header(sync_hash)?`

Reply:
replacing with `self.get_block`